src/e2e/app/cli/common/src/kamu_api_server_client.rs

```diff
@@ -8,13 +8,28 @@
 // by the Apache License, Version 2.0.
 
 use internal_error::{InternalError, ResultIntoInternal};
-use reqwest::{StatusCode, Url};
+use kamu_core::MediaType;
+use reqwest::{Method, Response, StatusCode, Url};
 use serde::Deserialize;
 use tokio_retry::strategy::FixedInterval;
 use tokio_retry::Retry;
 
 ////////////////////////////////////////////////////////////////////////////////
 
+pub enum RequestBody {
+    JSON(serde_json::Value),
+    NDJSON(String),
+}
+
+////////////////////////////////////////////////////////////////////////////////
+
+pub enum ExpectedResponseBody {
+    JSON(String),
+    PLAIN(String),
```

Comment:
Why all caps names? Won't clippy complain about these?

Reply:
> warning: variant `json` should have an upper camel case name

Interestingly, I discovered by accident that rustc allows upper case names for enum elements as well

Reply:
Fixed